### ISSUE TITLE
Vlc gets launched.

### ISSUE THREAD

**Issue by Wizardlysense** [SecUpwN/Spotify-AdKiller#71]
Once audio appears, Vlc starts playing recent video or movie.

**Comment by dspjm** [comment]
It plays my local file once spotify starts and I have to kill vlc in the background(vlc didn't appear). Then I can play the sons from spotify as usual.

**Comment by frcake** [comment]
VLC Gets spawned periodically after 4-5 ad mutes ..

**Comment by xxqxpxx** [comment]
i confirm , same happened with me

**Comment by Apttypn** [comment]
Yes same issue on Linux, i compiled this to windows and i have no ads and no problems either :)

**Comment by dspjm** [comment]
@Apttypn another reason for me to switch to windows now T_T @

**Comment by TWNateLo** [comment]
My vlc will launch and show random pictures...
The title of player shows "hardware YUV support"

**Comment by metalshreds** [comment]
Me too, random pictures are shown on VLC after a few ad mutes and at times it will cause my CPU to spike to 100% until I kill vlc.

**Comment by Wargof** [comment]
I have the same problem also under Ubuntu 18.04, it seems to me that it didn't do it under Ubuntu 16.04.

I'll try to uninstall VLC to see if it prevents these damn windows from opening.

**Comment by ErikMN** [comment]
It seems like this problem still persist on Ubuntu 20.04
